content/specs/versions.md

```diff
@@ -32,7 +32,9 @@ weight: 4
 - gem `3.0.3`
 - gh `1.8.1`
 - git `2.28.0`
+- Google Cloud SDK `270.0.0`

```

Comment:
Why not add the command, `gcloud`? Or are there a set of commands that come with this?